build/ci/scripts/android-uitest-run.sh

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
+set -euo pipefail
+IFS=$'\n\t'
+
+# echo commands
+set -x
+
+export UNO_UITEST_SCREENSHOT_PATH=$BUILD_ARTIFACTSTAGINGDIRECTORY/screenshots/android
+export UNO_UITEST_PLATFORM=Android
+export UNO_UITEST_ANDROIDAPK_PATH=$BUILD_SOURCESDIRECTORY/testing/TestHarness/TestHarness/bin/Release/net8.0-android/android-x64/uno.platform.extensions.demo-Signed.apk
```

Comment:
The application id is here https://github.com/unoplatform/uno.extensions/blob/main/testing%2FTestHarness%2FTestHarness%2FTestHarness.csproj#L12

That should be the APK file name